lib/nuget-parser/nuspec-parser.ts

```diff
@@ -82,42 +80,43 @@ function parseNuspec(dep, targetFramework, sep) {
 
 function extractDepsForPlainGroups(rawDependency) {
   return _(rawDependency.group)
-    .filter(function (group) {
+    .filter((group) => {
       // valid group with no attributes or no `targetFramework` attribute
       return group && !(group.$ && group.$.targetFramework);
     });
 }
 
 function extractDepsForTargetFramework(rawDependency, targetFramework) {
   return rawDependency && _(rawDependency.group)
-    .filter(function (group) {
+    .filter((group) => {
       return group && group.$ && group.$.targetFramework &&
         targetFrameworkRegex.test(group.$.targetFramework);
     })
-    .map(function (group) {
+    .map((group) => {
       const parts = _.split(group.$.targetFramework, targetFrameworkRegex);
       return {
         framework: parts[1],
+        group,
         version: parts[2],
-        group: group,
       };
     })
     .orderBy(['framework', 'version'], ['asc', 'desc'])
-    .find(function (group) {
+    .find((group) => {
       return targetFramework.framework === group.framework &&
         targetFramework.version >= group.version;
     });
 }
 
 function extractDepsFromRaw(rawDependencies) {
-  const deps = [];
-  _.forEach(rawDependencies, function (dependency) {
-    if (dependency && dependency.$) {
-      const dep = new Dependency(dependency.$.id, dependency.$.version);
-      deps.push(dep);
+  const deps: dependency.Dependency[] = [];
```

Comment:
can we import `{Dependency}` to get rid of `dependency.Dependency`?

Reply:
yes